src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,42 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int STACK_FULL_VALUE = 11;

```

Comment:
This constant is related to STACK_MAX_CAPACITY . Changing only one leads to an error (inconsistency).